core/templates/calendar.html

```diff
@@ -1,29 +1,98 @@
-<!--<!DOCTYPE html>
-<html>-->
-    <link rel="stylesheet" type="text/css" href="\css\calendar.css">
+<!DOCTYPE html>
+<html>
+  <link rel="stylesheet" type="text/css" href="\css\reset.css">
+  <link rel="stylesheet" type="text/css" href="\css\calendar.css">

```

Comment:
Firefox doesn't handle `\` like Chrome does. You'll want to switch the `href` attribute to use `/` instead